packages/x-charts-pro/src/internals/plugins/useChartProZoom/useChartProZoom.ts

```diff
@@ -91,43 +92,62 @@ export const useChartProZoom: ChartPlugin<UseChartProZoomSignature> = ({
     };
   }, [store, paramsZoomData]);
 
-  // Add instance methods
-  const setIsInteracting = React.useCallback(
-    (isInteracting: boolean) => {
-      store.update((prev) => ({ ...prev, zoom: { ...prev.zoom, isInteracting } }));
-    },
+  // This is debounced. We want to run it only once after the interaction ends.
+  const removeIsInteracting = React.useMemo(
+    () =>
+      debounce(() =>
```

Comment:
Shouldn't we define the timeout for the debounce?

Reply:
I'll define one, it just has a default I used implicitly